### DISCUSSION TITLE
Library dependency on apps from Tencent

### DISCUSSION THREAD

**Discussion by jjensen554git** [doronz88/pymobiledevice3#921] [Q&A]
While trying out a command: "python -m pymobiledevice3 remote tunneld --usb" I received 2 popups for installation of libraries from Tencent. I declined usage of these libraries. Is there another source for the same functionality? Thx.

**Comment by doronz88** [comment]
Not currently. We require creation of a TUNTAP device. If you have any other signed driver to use for windows you're more then welcome to submit a PR with it.

Also see #818
